File: app/src/main/java/com/example/narthana/genpass/data/WordContract.java
package com.example.narthana.genpass.data;

import android.provider.BaseColumns;

/**
 * Created by narthana on 23/10/16.
 */

public final class WordContract
{
    // prevent accidental instantiation
    private WordContract() {}

    public final static class WordEntry implements BaseColumns
    {
        public static final String TABLE_NAME = "words";
        public static final String COLUMN_WORD = "word";
        public static final String COLUMN_LEN = "length";
    }
}

File: app/src/main/java/com/example/narthana/genpass/data/WordContract.kt
package com.example.narthana.genpass.data

import android.provider.BaseColumns

/**
 * Created by narthana on 23/10/16.
 */

object WordContract {
    class WordEntry: BaseColumns {
        companion object {
            val _ID = BaseColumns._ID
            val TABLE_NAME = "words"
            val COLUMN_WORD = "word"
            val COLUMN_LEN = "length"
        }
    }
}